docs/source/deep-learning/keras/quickstart/quickstart_keras_core.ipynb

```diff
@@ -0,0 +1,856 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "2800a1f7-80cf-47cf-83af-b117ae946a34",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Get Started with Keras 3.0 + MLflow\n",
+    "**Author:** Jesse Chan<br>\n",
+    "**Date created:** 2023/11/05<br>\n",
+    "**Last modified:** 2023/11/20<br>\n",
+    "\n",
+    "This tutorial is an end-to-end tutorial on training a MINST classifier with **Keras 3.0** and logging results with **MLflow**. It will demonstrate the use of `mlflow.keras_core.MLflowCallback`, and how to subclass it to implement custom logging logic.\n",
+    "\n",
+    "**Keras** is a high-level api that is designed to be simple, flexible, and powerful - allowing everyone from beginners to advanced users to quickly build, train, and evaluate models. **Keras 3.0**, or Keras Core, is a full rewrite of the Keras codebase that rebases it on top of a modular backend architecture. It makes it possible to run Keras workflows on top of arbitrary frameworks — starting with TensorFlow, JAX, and PyTorch."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "df7eac27-132c-4541-830a-19dec603607b",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "## Install Packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "2aa11b8d-beb9-442a-b6fc-1088daf72ca8",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mWARNING: You are using pip version 21.2.4; however, version 23.3.1 is available.\n",
+      "You should consider upgrading via the '/Users/jesse.chan/Desktop/mlflow/.venvs/mlflow-env/bin/python3 -m pip install --upgrade pip' command.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install -q keras-core mlflow jax jaxlib torch tensorflow"
```

Comment:
For some reason our sphinx lint complains about this line. Let's inline this with text block to bypass the warning in doc-build.